gcloud/streaming/transfer.py

```diff
@@ -647,11 +643,11 @@ def from_file(cls, filename, mime_type=None, auto_transfer=True, **kwds):
         """
         path = os.path.expanduser(filename)
         if not os.path.exists(path):
-            raise NotFoundError('Could not find file %s' % path)
+            raise ValueError('Could not find file %s' % path)
```

Comment:
We could remove the "look before you leap" and just let `open(path, 'rb')` raise an `OSError`.